homeassistant/helpers/update_coordinator.py

```diff
@@ -192,6 +192,11 @@ async def async_refresh(self) -> None:
         for update_callback in self._listeners:
             update_callback()
 
+    @callback
+    def async_reset(self) -> None:
```

Comment:
Can you explain the use case ?

Reply:
Considering a way to solve https://github.com/home-assistant/core/issues/43845. TL;DR: SimpliSafe's API seems to have a bug when sensors are updated within 15 seconds of the last arming/disarming of the alarm. This would allow me to avoid situations where a coordinator update happens to occur in that window.

Reply:
I likely don't understand the problem well, but from your description in the issue: Would it also work to handle this inside your update method? That is, pause updates to the sensor by any of: not actually calling to the api during that time, raising an error, or holding on to a previous update and returning it

Reply:
Great point, @allenporter – probably a better way to solve things. 👍